mmcv/ops/deform_conv.py

```diff
@@ -246,6 +288,19 @@ def forward(self, x, offset):
                       pad_w].contiguous()
         return out
 
+    def __repr__(self):
+        s = self.__class__.__name__
+        s += f'(in_channels={self.in_channels}, '
```

Comment:
The repr string is a little bit long here, in one line.  Suggest keeping only one argument in a line.